src/open_inwoner/accounts/views/cases.py

```diff
@@ -149,24 +190,66 @@ def get_context_data(self, **kwargs):
             context["case"] = {
                 "identification": case.identificatie,
                 "start_date": case.startdatum,
-                "end_date": case.einddatum if hasattr(case, "einddatum") else None,
+                "end_date": (case.einddatum if hasattr(case, "einddatum") else None),
                 "description": case.omschrijving,
-                "type_description": case_type.omschrijving
-                if case_type
-                else _("No data available"),
-                "current_status": statuses[-1].statustype.omschrijving
-                if statuses
-                and statuses[-1].statustype.omschrijving
-                in [st.omschrijving for st in status_types]
-                else _("No data available"),
+                "type_description": (
+                    case_type.omschrijving if case_type else _("No data available")
+                ),
+                "current_status": (
+                    statuses[-1].statustype.omschrijving
+                    if statuses
+                    and statuses[-1].statustype.omschrijving
+                    in [st.omschrijving for st in status_types]
+                    else _("No data available")
+                ),
```

Comment:
Not related to the task: I parenthesized the inline if/else statements for readability.